survey_percent_question/README.rst

```diff
@@ -0,0 +1,45 @@
+Features
+========
+
+This module was written to offert a new type of question. 
+You can now ask in a survey the repartition in % though a chosen number of option 
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
+
+Configuration
+=============
+
+No particular configuration to use this module
+
+Usage
+=====
+
+To use this module, you need to :
+- create a survey
+- create a page
+- create a question
+
+Contributors
+============
+
+* Vincent Vinet <vincent.vinet@savoirfairelinux.com>
+* Mathieu Boldireff <matheu.boldireff@savoirfairelinux.com>
+* Bruno JOLIVEAU <bruno.joliveau@savoirfairelinux.com>
+
+Credits
+=======
+
+* Module developed and tested with Odoo version 8.0
+* For questions, please contact our support services <support@savoirfairelinux.com>
+
+Maintainer
+==========
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
Just some nitpicking: there is a more recent Readme OCA template,  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst